lib/config/presets/internal/workarounds.ts

```diff
@@ -132,6 +132,7 @@ export const presets: Record<string, Preset> = {
           'adoptopenjdk',
           'openjdk',
           'java',
+          'java-jre',
```

Comment:
```suggestion
          'java-jre',
          'java-jdk',
```

Reply:
Maybe also 'sapmachine', then it doesn't need a separate PR
https://github.com/renovatebot/renovate/pull/18122